.github/workflows/appimage.yml

```diff
@@ -103,7 +103,106 @@ jobs:
         sudo --preserve-env=GOOGLE_APPLICATION_CREDENTIALS -u testbot bash -lc 'cd rdk/services/slam/builtin && sudo go test -v -run TestOrbslamIntegration'
 
     - name: Build appimage
-      run: sudo -u testbot bash -lc 'cd slam/slam-libraries && make BUILD_CHANNEL="latest" appimage'
+      run: sudo -u testbot bash -lc 'cd slam/slam-libraries && make BUILD_CHANNEL="latest" appimageorb'
+
+    - name: Authorize GCP Upload
+      uses: google-github-actions/auth@v0.4.3
```

Comment:
Switch to @v1 here rather than pinning an out-of-date exact version. Actually, maybe just double-check all the workflows while your at it for that. (I know this is used in other jobs in the workflow too.)

Reply:
Got everything in the slam repo but looks like camera-servers is out of date as well, I'll go ahead and make a PR for that

Reply:
I don't think it's critical enough to need a PR right away or anything. It was just more of a "while you're in here..."